controllers/argocd/deployment.go

```diff
@@ -43,7 +45,7 @@ func getArgoCDRepoServerReplicas(cr *argoprojv1a1.ArgoCD) *int32 {
 	return nil
 }
 
-// getArgoCDServerReplicas will return the size value for the argocd-server replica count.
+// getArgoCDServerReplicas will return the size value for the argocd-server replica count if it has been set in argocd CR. Otherwise, nil is returned if the replicas is not set in the argocd CR or replicas value is < 0. If Autoscale is enabled, the value for replicas in the argocd CR will be ignored.
 func getArgoCDServerReplicas(cr *argoprojv1a1.ArgoCD) *int32 {
```

Comment:
```suggestion
// getArgoCDServerReplicas will return the size value for the argocd-server replica count if it 
// has been set in argocd CR. Otherwise, nil is returned if the replicas is not set in the argocd CR or
// replicas value is < 0. If Autoscale is enabled, the value for replicas in the argocd CR will be ignored.
func getArgoCDServerReplicas(cr *argoprojv1a1.ArgoCD) *int32 {
```